### ISSUE TITLE
Video playback in Pannellium "Partial Panorama"

### ISSUE THREAD

**Issue by gokhanmar** [mpetroff/pannellum#984]
I want to create a virtual exhibition where video can be displayed on a 180 degree surface. If it is possible to play video on the "Partial Panorama" surface, I think it will be a solution for me. I do not have enough knowledge about coding, I created a merged html file from two different documents on your site, but I could not show the video. Can you help with this?
(The video size I want to show is 5120x720px)

**Comment by mpetroff** [comment]
Yes, the partial panorama and video functionality can be combined. Starting from the [video example](https://pannellum.org/documentation/examples/video/), you'll need to modify the Pannellum configuration; replace
```javascript
videojs('panorama', {
    plugins: {
        pannellum: {}
    }
});
```
with
```javascript
videojs('panorama', {
    plugins: {
        pannellum: {
            vaov: 180,
            haov: 360
        }
    }
});
```
to get started. Then configure the `vaov` and `haov` parameters to match the vertical angle of view and horizontal angle of view of your video, in degrees. Based on the aspect ratio of the video resolution you mentioned, I'm assuming that the video does not reach the zenith and nadir, so you'll want to decrease the `vaov` parameter to match. Note that the video needs to use the equirectangular projection for the display to be accurate.

**Comment by gokhanmar** [comment]
Thank you so much. I was able to play the video with sample codes, but I could not run the autoplay command. Firefox automatically starts the video but chrome doesn't start it automatically. What is the solution to this?

**Comment by gokhanmar** [comment]
is it also possible to autoplay in mobile browsers?

**Comment by mpetroff** [comment]
For it to work in Chrome, you need to set Video.js's autoplay option to `'muted'`, e.g.,
```javascript
videojs('panorama', {
    autoplay: 'muted',
    plugins: {
        pannellum: {
            vaov: 180,
            haov: 360
        }
    }
});
```
That will probably fix it on mobile as well.
